build/pom.xml

```diff
@@ -27,7 +27,7 @@
     <version.javax.inject>1</version.javax.inject>
     <version.junit>4.12</version.junit>
     <version.mockito_all>1.10.19</version.mockito_all>
-    <version.testng>6.9.10</version.testng>
+    <version.testng>6.11</version.testng>
```

Comment:
This needs to be changed to `6.12`. @dipak-pawar can you give me push rights to your repo or to this branch?